client/components/DescriptionModule.js

```diff
@@ -44,54 +44,62 @@ const DescriptionModule = ({ gameid }) => {
 
   // On component mount, fetch relevant game information for component for a particular gameid (which is passed as props on mount)
   useEffect(() => {
-    fetchGamePhoto(gameid)
-      .then(res => {
-        if (res.error) {
-          throw new Error(res.error);
-        }
-        setGameThumbnail(res.url);
-      })
-      .catch(e => {
-        console.error(e);
-        // On fetchGamePhoto promise rejection, set default game thumbnail
-        setGameThumbnail('https://steamcdn-a.akamaihd.net/steam/apps/289070/header.jpg');
-      });
+    // fetchGamePhoto(gameid)
+    //   .then(res => {
+    //     if (res.error) {
+    //       throw new Error(res.error);
+    //     }
+    //     console.log(res.url);
+    //     setGameThumbnail(res.url);
+    //   })
+    //   .catch(e => {
+    //     console.error(e);
+    //     // On fetchGamePhoto promise rejection, set default game thumbnail
+    //     setGameThumbnail('https://steamcdn-a.akamaihd.net/steam/apps/289070/header.jpg');
+    //   });
+    setGameThumbnail('https://steamcdn-a.akamaihd.net/steam/apps/289070/header.jpg');
 
     fetchGameInfo(gameid)
       .then(res => {
         if (res.error) {
+          console.log('ddd1')
           throw new Error(res.error);
         }
         setGameInfo(res);
       })
       .catch(e => {
+        console.log('ddd2')
         // On fetchGameInfo promise rejection, keep initial game info state
         console.error(e);
       });
 
-    fetchAllGameReviews(gameid)
-      .then(res => {
-        if (res.error) {
-          throw new Error(res.error);
-        }
-        setAllReviews(res);
-      })
-      .catch(e => {
-        // On fetchAllGameReviews promise rejection, keep initial reviews state
-        console.error(e);
-      });
+    // fetchAllGameReviews(gameid)
+    //   .then(res => {
```

Comment:
Similarly, I mentioned this to @nightlaro too, but

It might be worth checking with @GrumpyCat0709 to make sure this service is going to be available and verify that it is compatible - this may have been  commented out by Christina because I took too long to provide this service for her, and she ended up generating the data herself.

Theoretically, this data feed will be available, but by the looks of this legacy code, it might not be parsing correctly.

Check the App & Service plan vs this module's parsing routine -

A request like this:
http://[SERVER]:3001/api/reviews/[PRODUCT_ID] *

should return data like this:
/1: {"summary":"Mixed","percent":64,"positive":3750,"negative":2059,"total":5809}

Reply:
It was commented by me to make the ui work since the service is not hosted yet. Once it's up I am going to change it recieve that actual data.